pyproject.toml

```diff
@@ -135,8 +160,10 @@ extend = "ruff-shared.toml"
 
 [tool.ruff.lint.isort]
 known-first-party = [
-    "controller",
-    "rubin.nublado",
+    "src/nublado",
+    "rubin-nublado-authenticator",
+    "rubin-nublado-client",
+    "rubin-nublado-spawner",
```

Comment:
All of this change is incorrect and I think this is why there are all the incorrect formatting changes. This is a list of module names, not a list of directory paths, so it needs to be `nublado`, `rubin.nublado`.